test_scripts/API/LastMileNavigation/GetWayPoints/004_GetWayPoints_UNSUPPORTED_RESOURCE_Navi_interface_is_not_avaliable.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Get%20Destination_and_Waypoints.md
+-- Item: Use Case 3: Exception 3: Navigation interface is not available on HMI
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid and allowed by Policies GetWayPoints_request to SDL and navigation interface is not available on HMI
+-- SDL must:
+-- 1) respond "UNSUPPORTED_RESOURCE, success:false" to mobile application and not transfer this request to HMI
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonLastMileNavigation = require('test_scripts/API/LastMileNavigation/commonLastMileNavigation')
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Local Functions ]]
+local function getHMIParams()
```

Comment:
@LitvinenkoIra , the same suggestion to rename e.g. disableNavigationIsReadyResponse